scripts/guides_master.py

```diff
@@ -23,6 +23,18 @@
             "path": "custom_image_augmentations",
             "title": "Custom Image Augmentations with BaseImageAugmentationLayer",
         },
+        {
+            "path": "semantic_segmentation_deeplab_v3_plus",
+            "title": "Semantic Segmentation with KerasCV",
+        },
+        {
+            "path": "coco_metrics",
```

Comment:
I'm not sure if we want to publish these other 2 right now -- I'm not certain of the state they are in, and at least for the COCO one we already discuss this in the OD guide.

Reply:
got it! I'll remove them